Monika After Story/game/script-topics.rpy

```diff
@@ -14435,3 +14435,61 @@ label monika_language_nuances:
     m 1tsu "So good day to you, [player].{w=0.3} {nw}"
     extend 1hub "Ahaha~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_architecture",
+            category=['misc'],
+            prompt="Architecture",
+            random=True
+        )
+    )
+
+label monika_architecture:
+    m 1esa "Hey, [player]! You know, I think there's this one major art branch that we've been missing to talk about all the time..."
+    m 3eub "Architecture!"
+    m 3esa "I have been reading a bit about it lately, so I find it quite interesting to discuss..."
+    m 1eua " If you come to think of it, architecture is one of the most common kinds of art in an everyday life."
+    m 1tub "Unless you live in a complete wasteland of some kind...{w=0.5}{nw}"
+    extend 1hub " Ahaha~"
+    m 1eka "But really, I am fascinated by how humanity tend to turn every craft into an art.{w=0.5}{nw}" 
+    extend 1eua " I think architecture is the greatest example of it."
+    m 1eub "Architecture can also tell a lot about the culture of a certain country it is located in: different monuments, statues, historical buildings, towers..."
+    m 1eua "I think it makes it even more exciting to explore the place you're visiting."
+    m 1rka "It is also important to place the buildings in the most convinient way for people to use, which is a tough task to deal with, too."
+    m 1euc "However, if you prefer viewing architecture only from the art perspective, some modern tendencies may disappoint you."
+    m 1ruc "Nowaday architecture focuses more on getting things done in a more practical way."
+    m 1euc "In my opinion, it can be either good or bad for multiple reasons." 
```

Comment:
Seems a bit awkwardly phrased. "that can be both good and bad" might be better.
You might also wanna put a comma between that and the "for multiple reasons", though I'm not 100% sure on that.